packages/block-library/src/latest-posts/edit.js

```diff
@@ -173,7 +176,14 @@ class LatestPostsEdit extends Component {
 				>
 					{ displayPosts.map( ( post, i ) =>
 						<li key={ i }>
-							<a href={ post.link } target="_blank">{ decodeEntities( post.title.rendered.trim() ) || __( '(Untitled)' ) }</a>
+							<a href={ post.link } target="_blank">
+								<RawHTML>
+									{
+										post.title.rendered.trim() ||
+										__( '(Untitled)' )
```

Comment:
Depending whether we consider translation as trusted strings†, it would be unsafe to allow this to be inserted in `RawHTML`

> If the latter is unavoidable, and since translations should not be considered trusted strings, be sure to sanitize the result before echoing.

https://codex.wordpress.org/I18n_for_WordPress_Developers#HTML

† I've heard conflicting reports that they are "safe". In this case, it seems easy enough to just consider whether the title exists before deciding to output as `RawHTML`.

```jsx
{
	post.title.rendered.trim() ?
		<RawHTML>{ post.title.rendered }</RawHTML> :
		__( '(Untitled)' )
}
```